### DISCUSSION TITLE
are Alltalk's generated sound files stored somewhere?

### DISCUSSION THREAD

**Discussion by joshgura** [erew123/alltalk_tts#341] [General]
just wondering, if Alltalk is generating some kind of audio file to be played, are these saved somewhere? like in a temp folder or something? because sometimes they end up sounding funny and it would be nice to save them. and if not, can they potentially be saved as they are playing with some kind of config tweak?

**Comment by erew123** [comment]
Please see the built in documentation:

![image](https://github.com/user-attachments/assets/cce28737-0b5d-4b54-ba7c-1490c91fa992)

**Reply by joshgura** [reply]
what does a typical alltalk tts wave output file formatting look like? i could try to search my system for a matching pattern because the way i figure, they have to be going somewhere unless they're just being passed directly to dev null.

ok, in the alltalk web configurations, the outputs are going to extensions/alltalk_tts/outputs/

i ran the demo using the alltalk browser configuration panel and the wav got saved to alltalk_tts/outputs. but this isn't happening when i use koboldcpp as the frontend, and i'm running kobold from a standalone executable. maybe kobold is receiving the file and dropping it somewhere. so maybe this is outside the scope of alltalk's purview.

running kobold's webui in brave/chromium by the way, looked in the cache after clearing it, generating some sounds, and nothing's being placed there either. I'll keep hunting and get back to you if I find where these wavs are going.
